camo/camo.d.ts

```diff
@@ -0,0 +1,139 @@
+// Type definitions for camo v0.11.4
+// Project: https://github.com/scottwrobinson/camo
+// Definitions by: Lucas Matías Ciruzzi <https://github.com/lucasmciruzzi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "camo" {
+
+	type TypeOrArray<Type> = Type | Type[];
+
+	/**
+	 * Supported type constructors for document properties
+	 */
+	export type SchemaTypeConstructor =
+		TypeOrArray<StringConstructor> |
+		TypeOrArray<NumberConstructor> |
+		TypeOrArray<BooleanConstructor> |
+		TypeOrArray<DateConstructor> |
+		TypeOrArray<ObjectConstructor> |
+		TypeOrArray<ArrayConstructor>;
+
+	/**
+	 * Supported types for document properties
+	 */
+	export type SchemaType = TypeOrArray<string | number | boolean | Date | Object>;
+
+	/**
+	 * Document property with options 
+	 */
+	export interface ISchemaTypeOptions<Type> {
```

Comment:
please remove `I` prefix from interface.
from http://www.typescriptlang.org/Handbook#writing-dts-files 

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.